stable/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 5.0.26
+version: 5.0.27
```

Comment:
@granular-ryanbonham 
Could you rebase to 5.1.3+ please?

Reply:
Rebased